docs/content/projects/cat.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "OSCAL CAT"
+description: "OSCAL Catalog Authoring Tool (CAT) - import, edit, and export OSCAL Catalogs from the browser"
+heading: "CAT: OSCAL Catalog Authoring Tool"
+toc:
+  enabled: true
+usabanner: true
+---
+
+## What is CAT?
+
+The OSCAL Catalog Authoring Tool (CAT) enables users to create, scope, tailor, control catalogs in [the OSCAL JSON format](https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/).
+
+## How does CAT work, is it right for me?
+
+CAT is designed in a convenient, web-based form factor and user-friendly interface. Users can quickly focus on what matters: creating or customizing their control catalog within the familiar experience of their preferred web browser. The responsive user interface allows users to manipulate controls and a resulting control catalog without complete understanding in OSCAL's information and data models in JSON. The focus is the content.
+
+CAT is a self-contained web application. The user downloads the application's content from the server upon first load, and then all subsequent catalog edits and processing is performed within the user's preferred browser. More advanced users can run the application locally on their own computer or an isolated network without internet access. (If you are interested in these more advanced use cases, [let the team know through GitHub](#how-do-i-request-support-with-my-own-deployment-of-oscal-cat-report-bugs-or-file-features).)
+
+## What are CAT's features? What can I do with catalogs?
+
+At this time, CAT enables users with a variety of features to manipulate a control catalog.
+
+- Use Author Mode to:
+  - create an OSCAL catalog from scratch.
+  - create an OSCAL catalog from [NIST SP 800-53 controls](https://csrc.nist.gov/publications/detail/sp/800-53/rev-5/final) and [SP 800-53B baseline defaults](https://csrc.nist.gov/publications/detail/sp/800-53b/final).
```

Comment:
Can only create a profile through selection and tailoring.

Reply:
Will edit to match this intent.